cmd/must-gather-clean/version.go

```diff
@@ -1,21 +1,16 @@
 package main
 
 import (
-	"fmt"
-
+	version "github.com/openshift/must-gather-clean/pkg/version"
 	"github.com/spf13/cobra"
 )
 
-const (
-	VERSION = "v0.0.1"
-)
-
 // versionCmd represents the version command
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print the version of the tool",
 	Run: func(_ *cobra.Command, _ []string) {
-		fmt.Println(VERSION)
```

Comment:
Example output:
```
Version: pkg.Version{Version:"v0.0.1-0-g6ce19dc", GitCommit:"6ce19dc", BuildDate:"2021-09-07T12:08:00Z", GoOs:"linux", GoArch:"amd64"}
```
`v0.0.1-0-g6ce19dc`
* tag v0.0.1
* 0 commits ahead of v0.0.1
* commit SHA

Reply:
https://git-scm.com/docs/git-describe#_examples